packages/react-vapor/src/components/childForm/examples/ChildFormExamples.tsx

```diff
@@ -3,6 +3,7 @@ import {ChildFormWithRadiosExamples} from './ChildFormWithRadiosExample';
 import {SimpleChildFormExample} from './SimpleChildFormExample';
 
 export class ChildFormExamples extends React.Component<any, any> {
+    static description = 'Child Forms are used to make accessible more controls when an option is selected in a form.';
     render() {
         return (
             <div className="mt2">

```

Comment:
This example is pretty nasty, because if you look in the sidenav, we have an entry for `ChildForm`, `SimpleChildFormExample` and `ChildFormWithRadiosExamples`. I would suggest to simply copy paste the content of `SimpleChildFormExample` and `ChildFormWithRadiosExamples` directly inside the `ChildFormExamples` file and delete those. This way we wouldn't have 3 nav entries for the same thing.